Doc/library/idle.rst

```diff
@@ -502,6 +502,11 @@ or immediately run an existing file before editing.
 Python Shell window
 ^^^^^^^^^^^^^^^^^^^
 
+With IDLE's Shell, one enters, edits, and recalls complete statements.
+Most consoles and terminals only work with a single physical line at a time.
+If one pastes more that one statement into Shell, Shell ignores anything
```

Comment:
This is not true. When I paste more that one statement into Shell, I get an error.
```
>>> print(1)
print(2)

SyntaxError: multiple statements found while compiling a single statement
```

Reply:
Right.  The extra is only ignored until one hits 'Enter' ;-).  Fixed, with an added note that one can edit before hitting Enter.